bls12_377/src/fields/tests.rs

```diff
@@ -24,10 +24,7 @@ generate_field_serialization_test!(bls12_377; fq2; fq6; fq12;);
 
 #[test]
 fn test_fq_repr_from() {
-    assert_eq!(
-        BigInteger384::from(100),
-        BigInteger384([100, 0, 0, 0, 0, 0])
-    );
+    assert_eq!(BigInteger384::from(100), BigInt::new([100, 0, 0, 0, 0, 0]));
```

Comment:
I have an upcoming PR that adds `From` impls from u32, u16, u8, so I would change this to 

```suggestion
    assert_eq!(BigInteger384::from(100u64), BigInt::new([100, 0, 0, 0, 0, 0]));
```